pytest_tests/testsuites/object/test_object_lock.py

```diff
@@ -119,6 +123,15 @@ def locked_storage_object(
 
 @pytest.mark.grpc_object_lock
 class TestObjectLockWithGrpc(ClusterTestBase):
+
+    @allure.title("Docker compose restart containers with new env file")
```

Comment:
It restarts the whole cluster, right?

Reply:
> It restarts the whole cluster, right?

I fixed the feature name and description - only storage nodes are restarted